pi-manage

```diff
@@ -1319,6 +1281,249 @@ def import_tokens(file, tokenrealm=None):
+                else:
+                    action_str = "Updated"
+            if conftype == "policy":
+                r = set_policy(name,
+                               action=config.get("action"),
+                               active=config.get("active", True),
+                               adminrealm=config.get("adminrealm"),
+                               adminuser=config.get("adminuser"),
+                               check_all_resolvers=config.get(
+                                   "check_all_resolvers", False),
+                               client=config.get("client"),
+                               # condition=policy.get("condition"),
+                               realm=config.get("realm"),
+                               resolver=config.get("resolver"),
+                               scope=config.get("scope"),
+                               time=config.get("time"),
+                               user=config.get("user"))
+            elif conftype == "resolver":
+                resolvertype = config.get("type")
+                data = config.get("data")
+                # now we can create the resolver
+                params = {'resolver': name, 'type': resolvertype}
+                for key in data.keys():
+                    params.update({key: data.get(key)})
+                r = save_resolver(params)
+            elif conftype == "event":
+                r = set_event(name, config.get("event"), config.get("handlermodule"),
+                              config.get("action"),
+                              conditions=config.get("conditions"),
+                              ordering=config.get("ordering"),
+                              options=config.get("options"),
+                              active=config.get("active"),
+                              position=config.get("position", "post"),
+                              id=event_id)
+
+            print("{0!s} {1!s} {2!s} with result {3!s}".format(action_str, conftype, name, r))
+
+
+@config_export_manager.option("--print_passwords", "-p", action="store_true",
+                              help="Print the passwords used in the resolver configuration. "
+                                   "This will overwrite existing passwords on import.")
+@config_export_manager.option("--archive", "-a", action="store_true",
+                              help="Compress the created config-backup as tar.gz archive")
+@config_export_manager.option("--stdout", "-s", action="store_true",
+                              help="Print the configuration also to stdout.")
+@config_export_manager.option("--directory", "-d", action="store_true",
+                              help="Directory where the backup will be stored.")
+def fullexport(directory=None, stdout=False, archive=False, print_passwords=False):
+    """
+    This option exports resolvers, policies and event handlers to standard output or to a file and optionally
+    compresses them as tar.gz archive
+    """
```

Comment:
Maybe we should also explicitly note that the main configuration will not be exported.
In the long run we should implement that as well.

Reply:
we will change things gradually. More sections of the privacyIDEA configuration can be added later. I think an explicit note is not necessary